.github/workflows/check-standalone-bayesian-optimization.yml

```diff
@@ -0,0 +1,31 @@
+name: Check instantiation of bayesian optimization on python 3.8
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install -r requirements-gpsearchers.txt --upgrade pip
+        python -m pip install typing_extensions
```

Comment:
Is this one needed? (It should be included in 3.8 AFAIK)

Reply:
As far as I've tested in my local repo, there was an error resulted from missing package of typing_extensions

Reply:
Oh I see, it is because we have a typing_extensions import left-over (should be removed and then we can ditch this dependency). Lets keep it that way for this PR, I will fix it in another one.